admin_manual/release_notes.rst

```diff
@@ -107,9 +110,15 @@ Home folder rule is enforced in the user_ldap application in new ownCloud instal
 
 The Calendar and Contacts apps have been rewritten and the CalDAV and CardDAV backends of these
 apps were merged into ownCloud core. During the upgrade existing Calendars and Addressbooks
-are automatically migrated. As a fallback for failed upgrades or an option to test a migration
-``dav:migrate-calendars`` and/or ``dav:migrate-addressbooks`` scripts are available via the
-``occ`` command. See :doc:`configuration_server/occ_command`.
+are automatically migrated (except when using the the ``IMAP user backend``). As a fallback
+for failed upgrades, when using the ``IMAP user backend`` or an option to test a migration
+``dav:migrate-calendars`` and/or ``dav:migrate-addressbooks`` scripts are available
+(**only in oC 9.1**) via the ``occ`` command. See :doc:`configuration_server/occ_command`.
```

Comment:
You're absolutely right, good eyes. Too much 9.0 / 9.1 in there, just have updated that part.